js/gdm/loginDialog.js

```diff
@@ -37,9 +37,9 @@ const Batch = imports.gdm.batch;
 const BoxPointer = imports.ui.boxpointer;
 const CtrlAltTab = imports.ui.ctrlAltTab;
 const GdmUtil = imports.gdm.util;
-const Layout = imports.ui.layout;
 const LoginManager = imports.misc.loginManager;
 const Main = imports.ui.main;
+const Monitor = imports.ui.monitor;
```

Comment:
As a heads up, if this commit ends up being rejected downstream, I would not want to keep it here either.
So perhaps you can just avoid it for now.